src/main/java/com/minecolonies/colony/buildings/AbstractBuildingWorker.java

```diff
@@ -113,24 +123,22 @@ public void writeToNBT(NBTTagCompound compound)
     /**
      * Set the worker of the current building
      *
-     * @param citizen       {@link CitizenData} of the worker
+     * @param citizen
+     *            {@link CitizenData} of the worker
      */
     public void setWorker(CitizenData citizen)
     {
-        if (worker == citizen)
-        {
-            return;
-        }
+        if (worker == citizen) { return; }
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS00122)